src/context.rs

```diff
@@ -473,11 +483,18 @@ impl Context {
                 *lock = Ratelimit::new(Duration::new(3, 0), 3.0);
             }
         }
+
+        if let Err(e) = self.create_gossip().await {
+            warn!(self, "{e}");
+        }
```

Comment:
- [ ] This is blocking, it delays my desktop startup time by 6 seconds with my 15 accounts. it should not block
- [ ] Also it might be better to do this when it's first used to risk to trigger any unexpected (to the user) firewall system dialogs at delta chat startup. (I don't know enough about iroh if it makes system calls that trigger this with `create_gossip` or not)